CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Fix `pycobertura diff` bug related to number of changed files. Thanks @ulasozguler
```

Comment:
Can we be more specific? Maybe reuse the wording used in the PR's description.

Reply:
Done.